inc/lock.class.php

```diff
@@ -112,7 +112,29 @@ static function showForItem(CommonDBTM $item) {
 
          }
 
-         $types = ['ComputerDisk', 'ComputerVirtualMachine'];
```

Comment:
why no just replace `ComputerDisk` by `itemDisk` ?

Reply:
bcause `$params` are not the same